packages/site-demo/content/product/components/date-picker/react/no-past-date-allowed.tsx

```diff
@@ -0,0 +1,26 @@
+import { Moment } from 'moment';
+import React from 'react';
+
+import { DatePicker } from '@lumx/react';
+
+const App = () => {
+    const LOCALE = 'en';
+
+    const now = new Date();
+    const minDate = new Date();
+    minDate.setDate(minDate.getDate() - 1);
+
+    const [datePicked, setDatePicked] = React.useState<Date | Moment | string>(now);
+
+    const handleDatePicked = (picked: Moment) => {
+        setDatePicked(picked);
+    };
+
+    return (
+        <div className="demo-grid">
+            <DatePicker value={datePicked} locale={LOCALE} onChange={handleDatePicked} minDate={minDate} />
```

Comment:
Could this work?
```suggestion
            <DatePicker value={datePicked} locale={LOCALE} onChange={setDatePicked} minDate={minDate} />
```

Reply:
Yes it would work but typescript will complain about type mismatching and the advantage of the handle function is that it made clear the fact that the onChange function return a Moment object and most of the time people will copy paste the snippet and that handle function is where they will put some code :)

Reply:
Or since moment is required in the end, we could simply put:
```tsx

    const now = moment();
    const [datePicked, setDatePicked] = React.useState(now);

    return (
        <div className="demo-grid">
            <DatePicker value={datePicked} locale={LOCALE} onChange={setDatePicked} minDate={now.substract(1, 'days')} />
```

Reply:
But I don't really minde either way